stock_inventory_verification_request/models/stock_slot_verification_request.py

```diff
@@ -81,23 +83,24 @@ def _get_involved_lines_and_locations(self):
             self._get_involved_lines_domain())
         return involved_moves, involved_lines
 
-    @api.one
+    @api.multi
     def action_confirm(self):
-        self.state = 'open'
-        involved_moves, involved_lines = \
-            self._get_involved_lines_and_locations()
-        self.involved_move_ids = involved_moves
-        self.involved_inv_line_ids = involved_lines
+        for rec in self:
+            rec.state = 'open'
```

Comment:
wouldn't be better a final `.write({...})`?

Reply:
I don't think this changes a lot. Most of the time this method is going to be called on a single record.